jar-plugin/src/main/groovy/com/github/jrubygradle/jar/internal/JRubyJarCopyAction.groovy

```diff
@@ -7,17 +7,19 @@ package com.github.jrubygradle.jar.internal
 */
 
 import com.github.jengelman.gradle.plugins.shadow.impl.RelocatorRemapper
+import com.github.jengelman.gradle.plugins.shadow.internal.GradleVersionUtil
```

Comment:
What is the purpose of relying on `GradleVersionUtil` ? (We should really try to avoid the internal APIs from Shadow)

Reply:
Philosophically I agree, relying on internal classes is icky, but we're already intimately wed to internal classes from the shadow plugin as is.

The only way around that would require upstream changes as far as I can tell.

Reply:
Looks like you have to merge it then. I had a brief look at GradleVersionUtil and it seems to do a couple of additional iffy things which cannot be handled by something like GradleVersion of Grolifant's Version directly.